src/components/input-dropdown/input-dropdown.component.ts

```diff
@@ -33,7 +33,7 @@ import { coerceCssPixelValue } from '@angular/cdk/coercion';
     ]
 })
 export class InputDropdownComponent<T> implements ControlValueAccessor, OnChanges, OnDestroy {
-    private _maxHeight: string;
+    maxHeightString: string;
```

Comment:
Could we keep this as `_maxHeight` we have a convention that variables that should be `private` but can't be because they are used in the template should still have the underscore prefix.

This is because if someone accesses the component via ViewChild for example, auto complete will suggest there is a property `maxHeightString` that they can access which we don't really want them to use as it is internal and they should be interacting with the input `maxHeight` instead, so the underscore prefix helps indicate that this value is still technically private.

I also think the description needs updated as this now accepts a string or number and can use any CSS unit.
![image](https://user-images.githubusercontent.com/20795331/63092604-2af26000-bf5a-11e9-871c-0111bddcdb11.png)

Reply:
I changed it.